receiver/statsdreceiver/go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsd
 go 1.18
 
 require (
+	github.com/lightstep/go-expohisto v1.0.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.62.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.62.0
```

Comment:
Hm, I think you will need a `replace` statement for this module added on this `go.mod`

Reply:
I see, so I probably haven't really tested the min and max support yet. I'll add a bit more testing of the DiffMetrics().

Reply:
See [3f95b9b](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12951/commits/3f95b9b2a1ae7eebdd962aec70294ba69a87c486). I'm concerned that these metrics test utilities are quite weak at testing and they could clearly be improved, but I also don't want to delay this work for want of testing infrastructure again (we did that once https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13428). Would you accept a new issue to cover improvements in testing utilities? (I've added TODOs to point out that the `MetricsDiff()` call here does not properly diff non-exponential histogram point Min and Max fields and the HasSum property).

Reply:
Yes, that is acceptable to me :+1:

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14934